11-Sophia-Pache/wk02/5-fri/atm/js/atm.js

```diff
@@ -0,0 +1,94 @@
+$(document).ready(function () { // this function will allow the user to DEPOSIT money in checking based on their input, adding the original value and the input value 
+    $('#checking-deposit').on('click', function () {
+        const depositVal = $('#checking-amount').val();
+        const balance = $('#checking-balance').text();
+        const number = (balance).slice(1);
+        const depositTotal = Number(number) + Number(depositVal);
+        $('#checking-balance').html(`$ ${depositTotal}`);
+    })
+})
+
+$(document).ready(function () { // this function will allow the user to WITHDRAW money from checking based on their input, subtracting the input value from the original value 
```

Comment:
```suggestion
```
You don't really need to have multiple "document ready" event listeners. I'd combine them into one.